ramls/schemas/fileDefinition.json

```diff
@@ -6,15 +6,43 @@
   "properties": {
     "id": {
       "description": "File id",
+      "type": "string",
+      "ref": "../common/uuid.json"
+    },
+    "fileName": {
+      "description": "File name",
       "type": "string"
     },
-    "path": {
+    "jobExecutionId": {
+      "description": "Job execution id",
+      "type": "string",
+      "ref": "../common/uuid.json"
+    },
+    "sourcePath": {
       "description": "Path to the file",
       "type": "string"
+    },
+    "createdDate": {
```

Comment:
can we use the metadata schema like in other modules? We may have to display who updated it and created it etc..

Reply:
This will also be another difference from other modules, we'll not have access to such data, of who updated it when